src/go/pt-mongodb-query-digest/main.go

```diff
@@ -407,32 +407,32 @@ func getTotalsTemplate() string {
 
 type lessFunc func(p1, p2 *stats.QueryStats) bool
 
-type multiSorter struct {
+type MultiSorter struct {
 	queries []stats.QueryStats
 	less    []lessFunc
 }
 
 // Sort sorts the argument slice according to the less functions passed to OrderedBy.
-func (ms *multiSorter) Sort(queries []stats.QueryStats) {
+func (ms *MultiSorter) Sort(queries []stats.QueryStats) {
 	ms.queries = queries
 	sort.Sort(ms)
 }
 
 // OrderedBy returns a Sorter that sorts using the less functions, in order.
 // Call its Sort method to sort the data.
-func OrderedBy(less ...lessFunc) *multiSorter {
-	return &multiSorter{
+func OrderedBy(less ...lessFunc) *MultiSorter {
```

Comment:
Sort, Len, Swap, Less are methods of the sort interface: https://pkg.go.dev/sort#Interface I suggest to leave name multiSorter as is and rename OrderedBy instead:

```suggestion
func orderedBy(less ...lessFunc) *multiSorter {
```

Reply:
Done 🍏